apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -192,6 +192,11 @@ function main() {
 
   var app = angular.module('teacherDashboard.controllers', []);
 
+  //helper function for using section-flow-2017 flag
+  const newFlow = function () {
+    return experiments.isEnabled('section-flow-2017');
+  };
+
```

Comment:
Note: Although this is fine, it could also have been written as a oneline
```js
const newFlow = () => experiments.isEnabled('section-flow-2017');
```